packages/lesswrong/components/sequences/TopPostsPage.tsx

```diff
@@ -653,12 +721,108 @@ const TopPostsPage = ({ classes }: { classes: ClassesType<typeof styles> }) => {
               {currentSortOrder === 'curated' ? sectionGrid : yearGrid}
             </div>
           </div>
+          <Divider margin={100}/>
+          <TopSpotlightsSection classes={classes} 
+            yearGroupsInfo={yearGroupsInfo} 
+            sectionsInfo={sectionsInfo} 
+            reviewWinnersWithPosts={reviewWinnersWithPosts} 
+          />
         </div>
       </AnalyticsContext>
     </>
   );
 }
 
+function TopSpotlightsSection({classes, yearGroupsInfo, sectionsInfo, reviewWinnersWithPosts }: {
+  classes: ClassesType<typeof styles>,
+  yearGroupsInfo: Record<ReviewWinnerYear, ReviewYearGroupInfo>,
+  sectionsInfo: Record<ReviewWinnerSectionName, ReviewSectionInfo>,
+  reviewWinnersWithPosts: PostsTopItemInfo[]
+}) {
+  const { SpotlightItem, LWTooltip } = Components;
+
+  const location = useLocation();
+  const { query } = location;
+
+  const years = Object.keys(yearGroupsInfo || {}).sort((a, b) => parseInt(b) - parseInt(a)).map((year) => parseInt(year)) as ReviewWinnerYear[]
+  const categories = Object.keys(sectionsInfo || {}).sort((a, b) => b.localeCompare(a)) as ReviewWinnerSectionName[]
+
+  const [year, setYear] = useState<ReviewWinnerYear|null>(query.year && reviewWinnerYearSet.has(parseInt(query.year)) ? parseInt(query.year) as ReviewWinnerYear : 2022)
+  const [category, setCategory] = useState<ReviewWinnerSectionName|null>(query.category && reviewWinnerSectionNamesSet.has(query.category) ? query.category as ReviewWinnerSectionName : "rationality")
+
+  useEffect(() => {
+    if (query.year) {
+      const element = document.getElementById('year-category-section');
+      if (element) {          
+        window.scrollTo({
+          top: element.getBoundingClientRect().top + window.pageYOffset - 400
+        });
+      }
+    }
+  }, [query.year]);
+
+  const filteredReviewWinnersForSpotlights = reviewWinnersWithPosts.filter(post => {
```

Comment:
I'm not sure how may posts you expect to be in `reviewWinnersWithPosts`, but if it's a lot you should probably wrap this in a `useMemo`.